utils/trivia.py

```diff
@@ -93,7 +96,11 @@ def end(self) -> int:
     def check(self, answer) -> int:
         # Return non-zero value to indicate failure
         # Return 0 to indicate success
-        return 0
+        if (self.qAsked.answer == answer)
```

Comment:
This looks sufficient for checking that the answer is an exact match.

If future work were to be considered, I would using some form of enhanced pattern matching or even providing multiple versions of the answer so that users do not have to enter the answer exactly as it is typed. In lieu of that, I would suggest at least converting the string to lowercase to do a case insensitive comparison so that users are not forced to also match capitalization.